### PR TITLE
Translate Frontend Utilities

### PR DESCRIPTION
**Pull request by stevietv**
#### Database Migration
NO

#### Description
Translate Frontend Utilities

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: frontend/src/Utilities/Date/formatDateTime.js [hunk 1]
```diff
@@ -10,15 +11,15 @@ function getRelativeDay(date, includeRelativeDate) {
   }
 
   if (isYesterday(date)) {
-    return 'Yesterday, ';
+    return `${translate('Yesterday')}, `;
```
Comment (markus101) [inline frontend/src/Utilities/Date/formatDateTime.js:14]: This is going to be another place where we need to have more context to translate properly. In this case, I think we just want to return the relative day without the comma then translate it again in `formatDateTime`, which could look something like: 

```
const relativeDay = getRelativeDay(date, includeRelativeDay);
const formattedDate = moment(date).format(dateFormat);
const formattedTime = formatTime(date, timeFormat, { includeMinuteZero: true, includeSeconds });

if (relativeDay) {
  return translate('{relativeDay}, ${formattedDate} ${formattedTime}', { relativeDay, formattedDate, formattedTime });
}

return translate('${formattedDate} ${formattedTime}', { formattedDate, formattedTime });
```

This way different locales that prefer the relative day at the end or time before the date can be handled better.

File: frontend/src/Utilities/Number/formatAge.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+
 function formatAge(age, ageHours, ageMinutes) {
   age = Math.round(age);
   ageHours = parseFloat(ageHours);
   ageMinutes = ageMinutes && parseFloat(ageMinutes);
 
   if (age < 2 && ageHours) {
     if (ageHours < 2 && !!ageMinutes) {
-      return `${ageMinutes.toFixed(0)} ${ageHours === 1 ? 'minute' : 'minutes'}`;
+      return `${ageMinutes.toFixed(0)} ${ageHours === 1 ? translate('Minute').toLowerCase() : translate('Minutes').toLowerCase()}`;
```
Comment (markus101) [inline frontend/src/Utilities/Number/formatAge.js:10]: If we don't need `Minute`, `Hour`, `Day` and the plural values capitalized I think we're better off not calling the `toLowerCase` after translating

**Review by markus101 — CHANGES_REQUESTED** [review]

File: frontend/src/Utilities/Number/formatAge.js (hunk 2 above)
Comment (stevietv) [reply frontend/src/Utilities/Number/formatAge.js:10]: we do have Day capitalized being used elsewhere, so I chose to just capitalize them all and lowercase them here.
Other wise we end up with a mix of `Day`  and `days`

Comment (stevietv) [reply frontend/src/Utilities/Number/formatAge.js:10]: hmm, thinking about it more, what if a particular locale doesn't want lowercase for their indication. I will create new keys for this

**Review by markus101 — APPROVED** [review]

**Pull request merged**